tutorials/applications/QuantumEvolutionKernel_Tutorial.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
```

Comment:
<p>*interactions, this <em>is</em> obtained...</p><p>*above, <em>without</em> interaction (red curve)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>